SilMock/Google/Service/Directory/Resource/Members.php

```diff
@@ -64,4 +70,43 @@ public function listMembers($groupKey, $optParams = [])
         }
         return $members;
     }
+
+    protected function validateGroupExists(string $groupKey): void
+    {
+        $mockGroupsObject = new Groups($this->dbFile);
+        $groupsObject = $mockGroupsObject->listGroups();
+        $groups = $groupsObject->getGroups();
+        $groupEmailAddresses = [];
+        foreach ($groups as $group) {
+            $groupEmailAddresses[] = $group->getEmail();
+        }
+        $uppercaseGroupEmailAddresses = array_map('mb_strtoupper', $groupEmailAddresses);
+        $uppercaseGroupEmailAddress = mb_strtoupper($groupKey);
+        if (! in_array($uppercaseGroupEmailAddress, $uppercaseGroupEmailAddresses)) {
+            throw new Exception(
+                sprintf(
+                    'Group %s does not exist',
+                    $groupKey
+                )
+            );
```

Comment:
Doesn't look like this would exceed line length if you squashed it to one line.

Reply:
It goes over 80 by a little. Formatting this way makes tweaks to the message or variables easier, in my opinion. No chance of accidentally changing something else beyond what was intended. Also, my PHPStorm cut-and-pastes this nicer than a more compressed version (sprintf and closing ")" on the line above and below.

Reply:
Didn't realize your limit was 80. Guess that way you can still work on a terminal like this: https://upload.wikimedia.org/wikipedia/commons/thumb/9/9f/DEC_VT100_terminal_transparent.png/1200px-DEC_VT100_terminal_transparent.png